packages/dashboard/tsconfig.json

```diff
@@ -15,7 +15,8 @@
     "noImplicitAny": true,
     "moduleResolution": "node",
     "preserveConstEnums": true,
-    "lib": ["es2017", "dom"],
+    "skipLibCheck": true,
+    "lib": ["es5", "es2015.promise", "dom"],
```

Comment:
@Raghurk I noticed there are several components in dashboard which use Promise. Do you require all consumers to provide a Promise polyfill (for IE 11 support)?